core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -695,6 +695,10 @@ public Expression implement(RexToLixTranslator translator, RexCall call,
 
         private NullAs negate(NullAs nullAs) {
           switch (nullAs) {
+          case IS_NOT_NULL:
+            return NullAs.IS_NULL;
```

Comment:
I guess this was unintended in the current PR

Reply:
yes this was not my change, not really sure how to remove it